app/views/admin/users/_reports.html.erb

```diff
@@ -0,0 +1,22 @@
+<div class="row my-3">
+  <div class="col-12">
+    <h2 class="d-inline">Recent Reports</h2>
+    <button type="button" data-toggle="collapse" data-target="#reactions-row" class="btn btn-secondary float-right">Toggle</button>
+  </div>
+  <div class="col-12 collapse" id="reactions-row">
+    <div class="list-group-flush">
+    <% unless @related_reports.empty? %>
+      <% @related_reports.each do |report| %>
+        <a href="<%= report.reported_url %>" class="list-group-item list-group-item-action">
```

Comment:
In my local app, `<%= report.reported_url %>` returns `nil` for some reports. For these reports, clicking on the link just reloads the current page. Is this acceptable?

Reply:
I'll look into this @msarit! I might just remove this link and/or adjust it to link to something else.